services/worker-manager/test/provider_aws_test.js

```diff
@@ -280,6 +287,7 @@ helper.secrets.mockSuite(testing.suiteName(), ['azure'], function(mock, skipping
   });
 
   suite('AWS provider - checkWorker', function() {
+    // TODO: Can someone fill these out and then I'll add tests for the lifecycle stuff here as well?
```

Comment:
plz to link to https://bugzilla.mozilla.org/show_bug.cgi?id=1577839

Reply:
I’d like this TODO to be fixed before we land this. It is ok if this PR gets landed after the 24 release I guess

Reply:
@owlishDeveloper can we write these tests now so that we know we're not breaking the provider with this change? Would it be better for you or me to write these tests?